src/sandstorm/identity.capnp

```diff
@@ -39,6 +39,13 @@ interface Identity {
 
   # TODO(someday): Public key info? Ability to seal messages / check signatures?
 
+  struct PowerboxTag {
+    # Tag to be used in a `PowerboxDescriptor` to describe an `Identity`.
+
+    identityId @0 :Data;
```

Comment:
It seems equally tricky or perhaps more tricky if it's in the powerbox tag, though.

For a method call, we can intercept and map at the realm gateway pretty easily.

Another possibility is to have `SandstormApi.getIdentityId(identity)`.

To deal with PowerboxTag we'd need to add a new system for intercepting and transforming powerbox descriptors in queries and fulfillments, which so far we don't have any need to do.

(Though admittedly it's unclear when you would ever want an Identity to be fulfilled from another grain, in which case all of this may be moot?)